app/models/person.rb

```diff
@@ -1,2 +1,3 @@
 class Person < ActiveRecord::Base
+  has_and_belongs_to_many :program
```

Comment:
Declining to change because the link between programs and people is not rich; there is nothing to be gained by has_many :through.